WordPress/gutenberg
# Add button text labels to site editor.

## Description


Partially addresses #15311. Like #24234, #24304 and #26135 did for the post editor, this adds an option to the site editor to show text labels instead of icons on interface buttons.

This might also be a good time to bring up #24370 again, as it would be convenient to be able to set this option across all editors at once.

## Testing Instructions





1. Open the site editor
2. In the "More tools & options" dropdown, enable the "Show button text labels" option
3. Check that buttons in the header and block toolbars now show text labels instead of icons

## Screenshots 

## Types of changes


New feature (non-breaking change which adds functionality)


## Checklist:
